boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -155,10 +165,80 @@ struct LoRaThingsPlus {
     temperature: &'static TemperatureDriver,
     humidity: &'static HumidityDriver,
     air_quality: &'static capsules_extra::air_quality::AirQualitySensor<'static>,
+    rng: Option<
+        &'static capsules_core::rng::RngDriver<
+            'static,
+            capsules_core::rng::Entropy32ToRandom<
+                'static,
+                capsules_extra::atecc508a::Atecc508a<'static>,
+            >,
+        >,
+    >,
     scheduler: &'static RoundRobinSched<'static>,
     systick: cortexm4::systick::SysTick,
 }
 
+#[cfg(feature = "atecc508a")]
+fn atecc508a_wakeup() {
+    let peripherals = (unsafe { PERIPHERALS }).unwrap();
+
+    peripherals.gpio_port[6].make_output();
+    peripherals.gpio_port[6].clear();
+
+    // The ATECC508A requires the SDA line to be low for at least 60us
```

Comment:
For reference, the timing (`t_WLO`) is specified on page 50 of [the datasheet](https://cdn.sparkfun.com/assets/learn_tutorials/1/0/0/3/Microchip_ATECC508A_Datasheet.pdf) and the requirement to hold SDA low for that long is on page 40.

The recommended "hack" for this is actually to send zero bits (e.g. `0x00`) at a slow enough baud rate to meet the same condition.

Reply:
We then need to lower the baud rate though

Reply:
Ack